docs/pages/dialer/dialer.yaml

```diff
@@ -718,26 +730,36 @@ components:
           items:
             - number:
                 type: string
-        direction:
-          type: string
-          enum:
-            - inbound
-            - outbound
         startTime:
           type: string
         connectedTime:
           type: string
         endTime:
           type: string
-        recordingUrl:
-          type: string
-          description: The URL of the recording
         createdTime:
           type: string
           description: The timestamp that the contact was created
         modifiedTime:
           type: string
           description: The timestamp the contact was last modified
+        recordingUrl:
+          type: string
+          description: The URL of the recording
```

Comment:
let's call out that this is supported for aircall and dialpad only? Same with the field below